src/transformers/pipelines/__init__.py

```diff
@@ -156,6 +160,7 @@
     "sentiment-analysis": "text-classification",
     "ner": "token-classification",
     "vqa": "visual-question-answering",
+    "text-to-audio": "text-to-speech",
```

Comment:
The pipeline can also be used as a text-to-audio pipeline!

Reply:
I would actually put it the other way, `text-to-audio` is more general which we tend to prefer.

That way when the audio procuced becomes music it still works.

Reply:
@osanseviero FYI do we already have a task name for this ?

Reply:
I will put it the other way around!

Reply:
So far we've said `text-to-speech` (https://huggingface.co/models?pipeline_tag=text-to-speech&sort=trending), we don't have a more general task name